src/test/groovy/org/gradle/android/CrossVersionOutcomeAndRelocationTest.groovy

```diff
@@ -542,6 +544,8 @@ class CrossVersionOutcomeAndRelocationTest extends AbstractTest {
         // Renamed from ToJar to ToDir
```

Comment:
I had to add some specific expectations as you can see below, the comment is probably obsolete then

Reply:
Yeah, let's remove this.  It's not really important and not technically correct.